--- checkpatch.pl	2019-03-05 19:04:46.000000000 +0900
+++ checkpatch.pl	2019-03-05 19:04:30.000000000 +0900
@@ -748,8 +748,8 @@
 }
 
 my $const_structs = "";
-read_words(\$const_structs, $conststructsfile)
-    or warn "No structs that should be const will be found - file '$conststructsfile': $!\n";
+read_words(\$const_structs, $conststructsfile);
+#    or warn "No structs that should be const will be found - file '$conststructsfile': $!\n";
 
 my $typeOtherTypedefs = "";
 if (length($typedefsfile)) {
