--- checkpatch-to-gerrit-json.py	2019-03-05 19:53:07.000000000 +0900
+++ checkpatch-to-gerrit-json.py	2019-03-05 19:44:03.000000000 +0900
@@ -7,11 +7,17 @@
 def main(fp=sys.stdin):
     """Convert checkpatch output into a gerrit-compatible JSON"""
     comments = {}
+    messages = ""
     while True:
         line = fp.readline()
+        if not line: break
         fileline = fp.readline()
-        if not fileline.strip():
+        if not fileline:
+            messages = messages + line
             break
+        if not fileline.strip():
+            messages = messages + line
+            continue
         while True:
             newline = fp.readline()
             if not newline.strip():
@@ -33,7 +39,7 @@
     if comments:
         output = {
             'comments': comments,
-            'message': 'Checkpatch %s' % line.strip()
+            'message': 'Checkpatch %s' % messages.strip()
         }
     else:
         output = {
